Makefile

```diff
@@ -22,15 +22,16 @@ BUILD_VERSION  ?= $(shell git describe --always --tags)
 
 MODIFIED = $(shell git diff-index --quiet HEAD || echo "-mod")
 
-SUBDIRS = dgraph
+#SUBDIRS = dgraph
```

Comment:
if this variable is not going to be used, it must be removed. It is not good practice to keep parts of code commented out.

Reply:
I agree.  This part of the Makefile

```
install:
	@(set -e;for i in $(SUBDIRS); do \
		echo Installing $$i ...; \
		$(MAKE) -C $$i install; \
	done)
```

looks like it at some point went through multiple subdirectories when installing.  Now it is only visiting the dgraph subdirectory.  If this logic is really not necessary anymore I can change this to just `$(MAKE) -C dgraph install`.

Reply:
I removed the for loop and the SUBDIRS variable.

Reply:
the test `t.go` binary & other local binaries (like `compose`) possibly get built through that `install`. I don't know if it's a good idea to remove it.

Reply:
As we discussed `make install` is not being removed, this change only removes the unnecessary for loop.  (Posting this comment in case someone reads this exchange later).